store/tikv/gcworker/gc_worker.go

```diff
@@ -240,7 +240,7 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 	return nil
 }
 
-// prepare checks required conditions for starting a GC job. It returns a bool
+// prepare checks preconditions for starting a GC job. It returns a bool
```

Comment:
Why change this file? Is it related to the dst issue?

Reply:
No. It has nothing to do with dst issue. I change it mainly becuase `GCoworker` already has `session` as its variable so we do not need access such info with passing it as parameter around. 

I found this on the way fixing dst, it should be done via another PR. I will be carefully next time.